### DISCUSSION TITLE
Critical issues that need fixing for next release?

### DISCUSSION THREAD

**Discussion by jonyo** [geodesicsolutions-community/geocore-community#168] [General]
@vicos59 since you've helped so much in testing, I wanted to check with you.  I was going to go ahead and release the next beta version, but wanted to check with you, are there any issues we should fix first?  As I recall, "most" of the issues are with PHP 8 / Smarty template upgrade related, so are generally quick for me to fix.

**Comment by vicos59** [comment]
The biggest issue overall, IMO, is the Facebook login.

Otherwise, it looks like the PayPal Pro fix disappeared when the source code was "built." 

BACKGROUND: In admin, when clicking on button to configure PayPal Pro payment gateway, the config area would not expand. I think you found a bug in a PEAR module and this solved the problem. However, after I built the latest source code and installed it on my server, the original problem reappeared.

Will have to check to see if it is still a problem in Beta-5.

Unofficial guess: since composer is used and PEAR is a library, perhaps the code change you made didn't get picked up by composer or was overwritten.

**Reply by jonyo** [reply]
Composer is not set up to update those PEAR library files, so that shouldn't be it.

There is a chance maybe it is using part libraries at the system level though. Or it could be another php error.

See if you see any errors in the response in the developer tools on the network tab (if you are familiar with that).

**Reply by jonyo** [reply]
I found it... It does not happen in PHP 8.1 because by that version, it simply ignores it and won't work lol... But if you tried it in PHP 7 it would cause a deprecation warning from one of the pear libraries that I missed.

Fixed in #175

**Reply by vicos59** [reply]
thanks, I'll check it out and let you know if it works on my server.
